lib/adapters/tool/testplane/index.ts

```diff
@@ -194,6 +197,31 @@ function getPluginOptions(config: Config): Partial<ReporterConfig> {
     return defaultOpts;
 }
 
+function createTool(configPath?: string): TestplaneWithHtmlReporter {
+    let tool!: TestplaneWithHtmlReporter;
+
+    for (const toolName of SUPPORTED_TOOLS) {
+        try {
+            // eslint-disable-next-line @typescript-eslint/no-var-requires
+            const Tool = require(toolName).default;
```

Comment:
I use `require` because in other way i should use `await import` in constructor (can't be async) or in static method `create`. But it is called inside plugin entry point, which is not async either.